lms/djangoapps/ccx/views.py

```diff
@@ -398,8 +400,14 @@ def ccx_student_management(request, course):
             unenroll_email(ccx, email, email_students=False)
     except ValidationError:
         log.info('Invalid user name or email when trying to enroll student: %s', email)
+        if email:
+            error_message = 'Could not find a user with name or email "{}"'\
+                .format(email)
```

Comment:
Please don't use "\" for line separation - edx prefers string continuation (e.g. http://stackoverflow.com/a/5437652) for very long string literals, but in this case, I would actually just space out the parameter:
```python
error_message = 'Could not find a user with name or email "{}"'.format(
    email
);
```